mmdet/datasets/openimages.py

```diff
@@ -0,0 +1,653 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+import csv
+import json
+import os.path as osp
+import pickle
+import warnings
+from collections import OrderedDict, defaultdict
+
+import numpy as np
+import torch.distributed as dist
+from mmcv.runner import get_dist_info
+from mmcv.utils import print_log
+
+from mmdet.core import eval_map
+from .builder import DATASETS
+from .custom import CustomDataset
+
+
+@DATASETS.register_module()
+class OpenImagesDataset(CustomDataset):
+    """Open Images dataset for detection."""
+
+    def __init__(self,
+                 label_description_file='',
+                 need_get_father=True,
```

Comment:
```suggestion
                 get_father_class=True,
```
BTW, does OpenImages use the name `father class`? Seems sexism here. `parent class` or `supercategory` would be better.